src/BiomeJsBinary.php

```diff
@@ -154,15 +154,24 @@ public static function getBinaryName(): string
         $os = strtolower(\PHP_OS);
         $machine = strtolower(php_uname('m'));
 
+        $isMusl = false;
+        ob_start();
+        phpinfo(INFO_GENERAL);
+        $buildinfo = ob_get_contents();
+        ob_end_clean();
+        if (preg_match('/--build=.*?-linux-musl/', $buildinfo)) {
+            $isMusl = true;
+        }
```

Comment:
Although interesting and quite clever, I'm not fan about this solution.

Can we assume that if we are running on Alpine, then we will have to download the Biomejs "musl" binary?

If yes, then we can check on file `/etc/alpine-release` presence.

WDYT?

Reply:
Checking whether we're in Alpine doesn't seem to solve the problem properly because :
- it would not be suitable for the other docker images (or environments) who are or will be based on musl.
- Alpine may decide to use the standard libc in the future

In fact, the core of the problem is the use of musl not Alpine.

This proposal relies on the presence of musl rather than the presence of Alpine, which avoids the situations mentioned.

Reply:
Hum, alright, that's what I thought 😕